src/libmeasurement_kit/dns/resolver.hpp

```diff
@@ -0,0 +1,146 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include <measurement_kit/common.hpp>
+#include <measurement_kit/dns.hpp>
+#include <measurement_kit/portable/sys/socket.h>
+#include <measurement_kit/portable/sys/types.h>
```

Comment:
No need to include stuff in `portable` directly because they're included by `common.hpp`